.github/ISSUE_TEMPLATE/1_7_0_beta1_feedback.yml

```diff
@@ -0,0 +1,175 @@
+      options:
+        - Did not test
+        - Tested, worked
+        - Tested, did not work / had problems
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Additional feedback / details on the state encryption
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Removed block
+        
+        - [Documentation](https://1-7-0-beta1.opentofu.pages.dev/docs/language/resources/syntax/#removing-resources)
+  - type: dropdown
+    attributes:
+      label: Did you test the removed block?
+      options:
+        - Did not test
+        - Tested, worked
+        - Tested, did not work / had problems
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Additional feedback / details on the removed block
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Testing feature
+        
+        - `run{}` blocks now allow referencing previous block's module outputs.
+        - `run{}` blocks now allow referencing the output from a previous run in variables. 
+        - `tofu test` now dumps the state file if it fails to clean up test resources.
+  - type: dropdown
+    attributes:
+      label: Did you test the testing feature changes?
+      options:
+        - Did not test
+        - Tested, worked
+        - Tested, did not work / had problems
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Additional feedback / details on the testing feature changes
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Built-in function changes
+        
+        - New function: [templatestring](https://1-7-0-beta1.opentofu.pages.dev/docs/language/functions/templatestring/)
+        - New function: [base64gunzip](https://1-7-0-beta1.opentofu.pages.dev/docs/language/functions/base64gunzip/)
+        - New function: [cidrcontains](https://1-7-0-beta1.opentofu.pages.dev/docs/language/functions/cidrcontains/)
+        - New function: [urldecode](https://1-7-0-beta1.opentofu.pages.dev/docs/language/functions/urldecode/)
+        - New function: [issensitive](https://1-7-0-beta1.opentofu.pages.dev/docs/language/functions/issensitive/)
+        - [nonsensitive](https://1-7-0-beta1.opentofu.pages.dev/docs/language/functions/nonsensitive/) no longer returns an error when the applied values are not sensitive.
+        - [templatefile](https://1-7-0-beta1.opentofu.pages.dev/docs/language/functions/templatefile/) now supports recursion up to a depth of 1024.
+  - type: dropdown
+    attributes:
+      label: Did you test the built-in functions?
+      options:
+        - Did not test
+        - Tested, worked
+        - Tested, did not work / had problems
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Additional feedback / details on the built-in functions
+
+  - type: markdown
+    attributes:
+      value: |
+        ## CLI changes
+        
```

Comment:
`tofu init now supports the -json flag for JSON output.` is missing from the above list.